src/main/java/com/gooddata/connector/Zendesk4ProcessExecution.java

```diff
@@ -58,6 +62,28 @@ public void setReload(final Boolean reload) {
         this.reload = reload;
     }
 
+    public Boolean getRecoverable() {
+        return recoverable;
+    }
+
+    /**
+     * tells if the newly started process should use recoverable feature
```

Comment:
Could you please add e.g. release version since when can these new fields be used?